pkg/report/template.go

```diff
@@ -32,21 +32,23 @@ var (
 
 // Generate create a Report on the filesystem.
 func Generate(dibReport Report, dag dag.DAG) error {
-	logrus.Infof("generating html report in the %s folder...", dibReport.GetRootDir())
-	if err := graphviz.GenerateGraph(&dag, dibReport.GetRootDir()); err != nil {
-		return fmt.Errorf("unable to generate graph: %w", err)
-	}
+	if len(dibReport.BuildReports) > 0 {
```

Comment:
You can use the early-return style here to avoid a level of indentation :

```suggestion
	if len(dibReport.BuildReports) == 0 {
		return nil
	}
	...
```

Reply:
Done